### PR TITLE
[BrowserKit] Add argument $serverParameters to `click()` and `clickLink()`

### LINKED ISSUE #18654: [BrowserKit] Add argument $serverParameters to click() and clickLink()
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#48841
| PR author(s) | @syl20b
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/18654

### CODE REVIEW

File: components/browser_kit.rst [hunk 1]
```diff
@@ -112,6 +112,28 @@ provides access to the link properties (e.g. ``$link->getMethod()``,
...
+can take a second, optional parameter, which is ``serverParameters``. This
+parameter allows to send additional information like headers when clicking
+on a link::
+
+    use Acme\Client;
+
+    $client = new Client();
+    $client->request('GET', '/product/123');
+
+    // Works both with `click()`...
```
Comment (OskarStark) [inline components/browser_kit.rst]:
```suggestion
    // works both with `click()`...
```

File: components/browser_kit.rst [hunk 2]
```diff
@@ -112,6 +112,28 @@ provides access to the link properties (e.g. ``$link->getMethod()``,
     $link = $crawler->selectLink('Go elsewhere...')->link();
     $client->click($link);
 
+The :method:`Symfony\\Component\\BrowserKit\\AbstractBrowser::click` and
+:method:`Symfony\\Component\\BrowserKit\\AbstractBrowser::clickLink` methods
+can take a second, optional parameter, which is ``serverParameters``. This
```
Comment (OskarStark) [inline components/browser_kit.rst]:
```suggestion
can take optional ``serverParameters``. This
```
Easier to read ?

**Review by OskarStark — APPROVED** [review]

File: components/browser_kit.rst (hunk 2 above)
Comment (alexandre-daubois) [reply components/browser_kit.rst]: Yes, I rephrased! :slightly_smiling_face:

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Pull request merged**